src/main/java/ay2021s1_cs2103_w16_3/finesse/ui/MainWindow.java

```diff
@@ -147,6 +192,58 @@ public void handleHelp() {
         }
     }
 
+    /**
+     * Opens the income window.
+     */
+    @FXML
+    private void handleIncome() {
+        panelLabel.setText("Income");
+        incomePanel = new IncomePanel(logic.getFilteredTransactionList());
+        transactionListPanelPlaceholder.getChildren().add(incomePanel.getRoot());
+        incomePanel.getRoot().toFront();
+
+        onIncome();
+    }
+
+    /**
+     * Opens the overview window.
+     */
+    @FXML
+    private void handleOverview() {
+        panelLabel.setText("Overview");
```

Comment:
`panelLabel` does not seem to correctly reflect the chosen tab.

![image](https://user-images.githubusercontent.com/5585517/95325035-34aff300-08d3-11eb-90ce-2cfb7cde74fd.png)

Note how `Income` is selected, yet the label displays `Expense`.

Reply:
This might be due to the bug found by @zhaojj2209 whereby switching tabs works only for the first time since the application was started.